### DISCUSSION TITLE
Oracle  data source

### DISCUSSION THREAD

**Discussion by vinayak12** [grafana/grafana-infinity-datasource#253] [Show and tell]
Kudos to you for developing a beautiful plugin.
Just wondering about any idea on Free oracle Plugins? oracle  DS is the biggest hindrance in using Grafana

**Comment by yesoreyeram** [comment]
Oracle plugin is the enterprise plugin. https://grafana.com/grafana/plugins/grafana-oracle-datasource/ I am not aware of any alternatives.

**Comment by andreadna** [comment]
https://github.com/albertowd/oracle-grafana
there is this one if you are still interested.

**Comment by keminc** [comment]
you can get free access to Oracle data from Grafana bt using [this](https://github.com/keminc/oracle2json_webserver) and Infinity plugin.
